test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/IntegrationTests/CodeGenerationIntegrationTest.cs

```diff
@@ -485,6 +485,7 @@ public void ViewWithNamespace_DesignTime()
             engine.Process(document);
 
             // Assert
+            AssertLineMappingsMatchBaseline(document);
```

Comment:
This should come between `AssertCSharpDocumentMatchesBaseline` and `AssertDocumentCompiles`

The line mappings being different is the *least* helpful thing you could use to diagnose a failure, so it should be later